crates/cheatnet/src/state.rs

```diff
@@ -267,6 +276,9 @@ pub struct TraceData {
 }
 
 pub struct CheatnetState {
+    pub cheated_execution_info_contracts: HashMap<ContractAddress, ExecutionInfoMock>,
```

Comment:
So CheatStatus is not needed here?

Reply:
`CheatStatus` is used in `ExecutionInfoMock`